### ISSUE TITLE
gamemodes.autoset.XXXXX isn't working with PEX

### ISSUE THREAD

**Issue by fultrim** [sunkid/WorldModes#10]
Bukkit - 1.01
Gamemodes - 2.5.2 (version from today)
BaseBukkit - ??? (version from today)

I have tried to configure the permission through inheritance in global perms, assigned directly in global, and assigned directly in world perms.  In every case the user doesn't get changed.

**Comment by sunkid** [comment]
Can you post your permissions? gamemodes.autoset.xxx is not a WorldModes permission node ;)

**Comment by fultrim** [comment]
Sunkid.  You are correct.  I intended to say worldmodes.autoset.XXX where X is survival or creative.

I also found the issue was with PEX inheritance.  The default rank has worldmodes.autoset.survival and then player and above were using worldmodes.autoset.creative.  However, PEX was giving both perms to player and above.  

Thus when the world loaded and the hook for autoset was called it would see both perms and then do nothing.  

I resolved this by breaking inheritance for the default group since that group was the only group not following the normal scheme.
